lib/components/molecules/Quote/Quote.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import React from 'react';
+import type { Node } from 'react';
+import styled from 'styled-components';
+import classnames from 'classnames';
+import styles from './Quote.style';
+
+type Props = {
+  className: string,
+  content: string,
+  contentSrc: string,
+  author?: string,
+  company?: string,
+};
+
+const Quote = ({ content, contentSrc, author, company, className, ...otherProps }: Props): Node => {
+  return (
+    <section {...otherProps} className={classnames('quote-wrap', className)}>
```

Comment:
Why does your Quote component makes use of a section?

Reply:
fixed.